<ui version="4.0" >
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>519</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>videocapturetestapp</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="video_image_label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>320</width>
         <height>240</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="pixmap" >
        <pixmap/>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="KPushButton" name="take_photo_button" >
       <property name="text" >
        <string>Take a Photo</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="help_button" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="quit_button" >
       <property name="text" >
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="video_tab_widget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="device_tab" >
      <attribute name="title" >
       <string>Device</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QLabel" name="device_label" >
             <property name="text" >
              <string>Device:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="input_label" >
             <property name="text" >
              <string>Input:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="standard_label" >
             <property name="text" >
              <string>Standard:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="KComboBox" name="device_combo_box" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip" >
              <string>Select the device to take video input from</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KComboBox" name="input_combo_box" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip" >
              <string>Select from multiple inputs provided by some devices</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KComboBox" name="standard_combo_box" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip" >
              <string>Select the video standard to use</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="options_tab" >
      <attribute name="title" >
       <string>Optio&amp;ns</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QCheckBox" name="adjust_brightness_combo_box" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Enable automatic brightness/contrast adjustment</string>
         </property>
         <property name="text" >
          <string>Adjust brightness/contrast</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="correct_colors_combo_box" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Enable automatic color correction</string>
         </property>
         <property name="text" >
          <string>Correct colors</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mirror_preview_check_box" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Show the video input mirrored</string>
         </property>
         <property name="text" >
          <string>Mirror preview</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="controls_tab" >
      <attribute name="title" >
       <string>Con&amp;trols</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="brightness_label" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Brightness:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QSlider" name="brightness_slider" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum" >
            <number>65535</number>
           </property>
           <property name="singleStep" >
            <number>256</number>
           </property>
           <property name="pageStep" >
            <number>4096</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="contrast_label" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Contrast:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QSlider" name="contrast_slider" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum" >
            <number>65535</number>
           </property>
           <property name="singleStep" >
            <number>256</number>
           </property>
           <property name="pageStep" >
            <number>4096</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="saturation_label" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Saturation:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QSlider" name="saturation_slider" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum" >
            <number>65535</number>
           </property>
           <property name="singleStep" >
            <number>256</number>
           </property>
           <property name="pageStep" >
            <number>4096</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="whiteness_label" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Whiteness:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QSlider" name="whiteness_slider" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum" >
            <number>65535</number>
           </property>
           <property name="singleStep" >
            <number>256</number>
           </property>
           <property name="pageStep" >
            <number>4096</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="hue_label" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Hue:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QSlider" name="hue_slider" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum" >
            <number>65535</number>
           </property>
           <property name="singleStep" >
            <number>256</number>
           </property>
           <property name="pageStep" >
            <number>4096</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
